src/main/java/com/nextstep/lotto/LottoInput.java

```diff
@@ -0,0 +1,26 @@
+package com.nextstep.lotto;
+
+import java.util.Scanner;
+
+public class LottoInput {
+
+	String BUY_LOTTO_MESSAGE = "구입금액을 입력해주세요.";
```

Comment:
접근제어자를 생략한 이유가 있을까요?

```suggestion
	private static final String BUY_LOTTO_MESSAGE = "구입금액을 입력해주세요.";
```